File: src/bot.ts
import { Bot, InlineKeyboard, webhookCallback } from "grammy";
import { chunk } from "lodash";
import express from "express";


// Create a bot using the Telegram token
export const bot = new Bot(process.env.TELEGRAM_TOKEN || "");

bot.command("id", (ctx) => {
  const chat = ctx.chat;
   if (chat?.type === "group" || chat?.type === "supergroup") {
    const groupId = chat.id;
    const formattedId = `\`${groupId}\``;
    ctx.reply(`Group Chat ID: ${formattedId}`, { parse_mode: "MarkdownV2" });
  } else {
    ctx.reply("This command can only be used in a group chat.");
  }
});


  bot.start();

File: src/bot.py
from telegram.ext import Updater, CommandHandler

# Define a function to handle the /start command
def start(update, context):
    context.bot.send_message(chat_id=update.effective_chat.id, text="Hello, I'm your bot!")

# Create an instance of the Updater class with your bot token
updater = Updater(token='process.env.TELEGRAM_TOKEN ', use_context=True)

# Get the dispatcher to register handlers
dispatcher = updater.dispatcher

# Register a command handler for the /start command
dispatcher.add_handler(CommandHandler('start', start))

# Start the bot
updater.start_polling()